e2e/cypress/integration/accessibility/accessibility_sidebar_spec.js

```diff
@@ -8,7 +8,7 @@
 // ***************************************************************
 
 // Stage: @prod @smoke
-// Group: @accessibility
+// Group: @accessibility @smoke
```

Comment:
I'm confused how do you plan to run this. 
If `@smoke` will be on "Group" metadata which I think makes sense, then the `@smoke` tags on "Stage" can be removed. With that, it could run as `node run_tests.js --stage='@prod' --group'@smoke'`
```suggestion
// Stage: @prod
// Group: @accessibility @smoke
```

Reply:
Oh, I just read your updated description with `node run_tests.js --stage='@smoke' --group='@smoke'`. However, passing `@smoke` on both args feels redundant to me.